tests/shakedown/shakedown/dcos/command.py

```diff
@@ -211,7 +210,7 @@ def attached_cli():
 
     The CLI setup command should be idempotent. So it is save to call this method multiple times.
     """
-    cmd = 'cluster setup {} --no-check --username={} --password={}'.format(dcos_url(), dcos_username(), dcos_password())
+    cmd = 'cluster setup {} --no-check '.format(dcos_url())
```

Comment:
It wouldn't work without `--username={} --password={}` when I tried it locally. I also _exported_ the `DCOS_USERNAME` and `DCOS_PASSWORD` env vars prior to the test.

Reply:
Which version did you use?